apps/chat/tasks.py

```diff
@@ -28,15 +29,17 @@ def periodic_tasks(self):
 
 
 @shared_task
-def send_bot_message_to_users(message: str, chat_ids: List[str], is_bot_instruction: bool):
+def send_bot_message_to_users(message: str, chat_ids: List[str], is_bot_instruction: bool, experiment_public_id: UUID):
     """This sends `message` to the sessions related to `chat_ids` as the bot.
```

Comment:
Why the experiment's public id? The feature that Delta's using, she'll need to manually create these periodic tasks and she only has the experiment's public ID at hand. It's sort of to accomodate that feature.